File: mcached/scala/delete/mcached_delete.scala
// ----------------------------------------------------------------
/*
	mcached_delete.scala

					Feb/05/2015

*/
// ----------------------------------------------------------------
object	mcached_delete
{
// ----------------------------------------------------------------
def main (args: scala.Array[String])
{
	println ("*** 開始 ***")

	val key_in = args(0)

	println (key_in)

	val server = "localhost"
	val port = 11211
	print (server + '\t')
	println (port)

	mcached_manipulate.delete_record_proc (key_in,server,port)

	println ("*** 終了 ***")
}

// ----------------------------------------------------------------
}
// ----------------------------------------------------------------

File: mcached/lua/delete/mcached_delete.lua
#! /usr/bin/lua
-- ----------------------------------------------------
--	mcached_delete.lua
--
--				Feb/13/2015
--
-- ----------------------------------------------------
require	('mcached_manipulate')
-- ----------------------------------------------------
print ("*** 開始 ***")

key_in=arg[1]

print (key_in)

local server_host = "localhost"
local server_port = 11211

mcached_delete_proc (server_host,server_port,key_in)

print ("*** 終了 ***")
-- ----------------------------------------------------
